plugin/evm/limitorders/limit_order_tx_processor.go

```diff
@@ -134,7 +137,12 @@ func (lotp *limitOrderTxProcessor) executeLocalTx(contract common.Address, contr
 		log.Error("HexToECDSA failed", "err", err)
 		return err
 	}
-	tx := types.NewTransaction(nonce, contract, big.NewInt(0), 5000000, big.NewInt(70000000000), data)
+	baseFeeEstimate, err := lotp.getUpdatedBaseFee()
```

Comment:
if this remains the same for a particular block, can we cache it for all txs made in a single block